packages/renderer/src/lib/recommendation/ExtensionBanner.svelte

```diff
@@ -13,7 +13,7 @@ let style: string | undefined = undefined;
 let hasBackground: boolean =
   !!banner.background && ((!!banner.background.dark && !!banner.background.light) || !!banner.background.gradient);
 
-onMount(() => {
+afterUpdate(() => {
```

Comment:
afterUpdate will be deprecated https://svelte-5-preview.vercel.app/docs/deprecations

Reply:
I think the update needs to occur when isDark is updated

Reply:
Yes I've pushed conversion to svelte5 which could also work (option 1)
Or just add `$: if (isDark) {
  changeStyle();
}` function which will be also called in `onMount()` (option 2)
@benoitf @axel7083